.github/workflows/docs-build.yml

```
@@ -42,15 +52,15 @@ concurrency:
 jobs:
   changelog-develop:
     name: Create changelog of FAF Develop
-    uses: ./.github/workflows/docs-changelog.yaml
+    uses: ./.github/workflows/docs-generate-changelog.yaml
```

actionlint (1.7.12)

[error] could not read reusable workflow file for "./.github/workflows/docs-generate-changelog.yaml": open /home/jailuser/git/.github/workflows/docs-generate-changelog.yaml: no such file or directory

(workflow-call)